README.md

```diff
@@ -68,16 +111,217 @@ you can also chain components names `react-component <ComponentName> <ComponentN
+
+    cd path/to/repo\ Folder
+
+Initialize Settings and Templates
+
+    react-component init -t
+
+This will create:
+
+    Repo Folder
+        | - .ccr
+            | - templates
+                | - component
+                |   | - component.dot
+                |
+                | - index
+                |   | - index.dot
+                |
+                | - style
+                |   | - style.css.dot
+                |
+                | - settings.json
+
+open `component/component.dot` and you should see something like this.
+
+        import React, { Component } from 'react';
+        {{? it.component.useCSS }}
+        import './{{= it.component.name }}.css';
+        {{?}}
+
+        class {{= it.component.name}} extends Component {
+          // constructor(props){
+            // super(props);
+            // this.state = {};
+          // }
+
+          // componentWillMount(){}
+          // componentDidMount(){}
+          // componentWillUnmount(){}
+
+          // componentWillReceiveProps(){}
+          // shouldComponentUpdate(){}
+          // componentWillUpdate(){}
+          // componentDidUpdate(){}
+
+          render() {
+            return (
+              <div></div>
+            );
+          }
+        }
+        export default {{= it.component.name}};
+
+> Note: All templates in the `.ccr/templates` folder are the default templates for create-components-react.
+
+`component/component.dot` is the default template to make each component file. You now can edit this file to make it look like anything you want as long as you follow the rules of javascript and doT. After you update this file and save it you can now use the `create` command and each component created will now take on the new look of the template file you updated.
+
+Example: edit `component/component.dot` to render a component that doesnt include any life cycle methods and always uses the constructor function. Now your file should look like this:
+
+        import React, { Component } from 'react';
+        {{? it.component.useCSS }}
+        import './{{= it.component.name }}.css';
+        {{?}}
+
+        class {{= it.component.name}} extends Component {
+          constructor(props){
+            super(props);
+            this.state = {};
+          }
+
+          render() {
+            return (
+              <div></div>
+            );
+          }
+        }
+        export default {{= it.component.name}};
+
+Now when you execute `react-components create App` it should make a directory like this:
+
+        Repo Folder
+        | - .ccr
+            | - ...
+        | _ App
+            | - App.js
+            | - index.js
+            | - style.css
+
+This doesnt look to different than the default version right? Dont be disappointed to fast. Lets take a look at `App.js`. This file is create from the `component/component.dot` so if we have all our configurations right, it should now look like this:
```

Comment:
This doesnt look to different -> This doesn't look too different
Dont be disappointed to fast -> Don't be disappointed too quickly
Lets -> Let's
This file is create -> This file is created 

For too vs to, I think if something is too much. Like it's too big, or too quiet. If it fits this case, too should be used.
Lets vs lets: let's is a contraction of let us, lets can be used for like being allowed to do something.